examples/helia-101/package.json

```diff
@@ -16,13 +17,18 @@
     "@chainsafe/libp2p-yamux": "^7.0.1",
     "@helia/http": "^2.0.0",
     "@helia/unixfs": "^4.0.0",
+    "@ipld/unixfs": "^3.0.0",
     "@libp2p/bootstrap": "^11.0.7",
     "@libp2p/identify": "^3.0.7",
     "@libp2p/tcp": "^10.0.8",
     "blockstore-core": "^5.0.2",
     "datastore-core": "^10.0.2",
     "helia": "^5.0.0",
-    "libp2p": "^2.1.6"
+    "ipfs-unixfs": "^11.2.0",
+    "ipfs-unixfs-exporter": "^13.6.1",
+    "ipfs-unixfs-importer": "^15.3.1",
+    "libp2p": "^2.1.6",
```

Comment:
No need to these 3 explicitly

Reply:
Sorry. This is my bad. libp2p was already in here because we import it directly in the 301 example.